<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Manjunath" failures="1" tests="3" name="Test" time="6.762" errors="0" timestamp="03 Oct 2021 14:59:28 GMT">
  <testcase classname="customAnnotation.Script" name="createContact" time="0.026"/>
  <testcase classname="customAnnotation.Script" name="updateContact" time="0.006">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at customAnnotation.Script.updateContact(Script.java:21)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- updateContact -->
  <testcase classname="customAnnotation.Script" name="deleteContact" time="0.002"/>
</testsuite> <!-- Test -->
